participants-schema.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Participants Schema",
+  "type": "object",
+  "definitions": {
+    "url": {
+      "type": "string",
+      "pattern": "^(http|https)://[A-Za-z0-9ÄÖÜäöüß\\-_\\./]+"
+    }
+  },
+  "properties": {
+    "name": {
+      "type": "string"
+    },
+    "pronouns": {
+      "type": "string"
+    },
+    "link": {
+      "$ref": "#/definitions/url"
+    },
+    "mastodon": {
+      "type": "string",
+      "pattern": "^(http|https)://[A-z0-9.\\-/]+/@[A-z0-9._%+\\-]+$"
+    },
+    "image": {
+      "$ref": "#/definitions/url"
+    },
+    "contact-via": {
+      "type": "string",
+      "pattern": "^(^[A-z0-9äöüÄÖUß_\\-\\.\\+]+@[A-z0-9äöüÄÖUß_\\-\\.]+\\.[A-Za-z]{2,}$|[\\+]?[(]?[0-9]{3}[)]?[-\\s\\.]?[0-9]{3}[-\\s\\.]?[0-9]{4,6})$"
```

Comment:
```suggestion
```
There is no specific format for this field.

Reply:
In template it's defines as "email or phone number", but I don't have a strong optinion. We maybe should add a minLength?

Reply:
I removed patternd and added a minLength condition